common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -429,15 +407,14 @@ nav.sequence-bottom {
   }
 }
 
-.xmodule_VerticalModule div.vert-mod > div ul.sequence-nav-buttons {
-  list-style: none !important;
-}
-
-.xmodule_SequenceModule nav.sequence-bottom ul li.next a,
-.xmodule_SequenceModule nav.sequence-bottom ul li.prev a {
+#seq_content {
 
-/*  &:focus {
-    outline: 0;
+  &:focus,
+  &:active {
+    outline: none;
   }
-*/
+}
+
+.xmodule_VerticalModule div.vert-mod > div ul.sequence-nav-buttons {
+  list-style: none !important;
```

Comment:
Do you really need the ``!important`` here? if so, I'd consider moving this to the ``_shame.scss`` partial with a note about where/why its done.